app/controllers/admin/configs_controller.rb

```diff
@@ -233,5 +256,10 @@ def brand_color_not_hex
       hex = params.dig(:site_config, :primary_brand_color_hex)
       hex.present? && !hex.match?(/\A#(\h{6}|\h{3})\z/)
     end
+
+    def valid_image_url(url)
+      valid_url = %r{\A(http|https)://([/|.|\w|\s|-])*.(?:jpg|gif|png)}
```

Comment:
**suggestion (non-blocking):** It would be great to move the regular expression into a constant so it doesn't get recompiled on every method invocation.

Reply:
TIL! Great call -- I've made this change.